### ISSUE TITLE
Error in running DB created in OrientDB 2.1-rc5 under OrientDB 2.1

### ISSUE THREAD

**Issue by PhantomYdn** [orientechnologies/orientdb#4756]
During startup I can observe the following exceptions:

```java
Caused by: java.lang.AssertionError
        at com.orientechnologies.orient.core.storage.impl.local.paginated.OPaginatedCluster.updateRecord(OPaginatedCluster.java:862)
        at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.doUpdateRecord(OAbstractPaginatedStorage.java:1943)
        at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.updateRecord(OAbstractPaginatedStorage.java:888)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.executeSaveRecord(ODatabaseDocumentTx.java:1901)
        ... 80 more
```

and later on a lot of exceptions like this:

```java
com.orientechnologies.orient.core.exception.OStorageException: Invalid value of TX counter
        at com.orientechnologies.orient.core.tx.OTransactionOptimistic.rollback(OTransactionOptimistic.java:175)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.commit(ODatabaseDocumentTx.java:2591)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx.commit(ODatabaseDocumentTx.java:2547)
```

**Comment by PhantomYdn** [comment]
It seems that the same issue exist for previous versions if run them under 2.1.
For freshly created DB everything is fine.

**Comment by laa** [comment]
Yes it is the same I close it.

**Comment by lvca** [comment]
@laa Does 2.1 has such problems with database created with previous versions?

**Comment by PhantomYdn** [comment]
Guys, any ideas how to fix dumps created under 2.1-rcX without export-import? Unfortunately, we have a couple of production DBs created under 2.1-rcX...

**Comment by laa** [comment]
Hi, I fixed this in 2.1.1 branch   but  it is simple to fix, disable assertions and you will never have this error.
